pubspec.yaml

```diff
@@ -29,7 +29,7 @@ dependencies:
   flutter_svg: ^0.23.0+1
   nstack:
     git:
-      url: git://github.com/nstack-io/flutter-sdk.git
+      url: https://github.com/nstack-io/flutter-sdk.git
       #ref: v0.2.5
```

Comment:
thanks 👍  while we're at it, let's also uncomment this and set the latest version `v0.5.1`

Reply:
Done!